components/demoplugin/Makefile

```diff
@@ -7,7 +7,9 @@ PLATFORMS ?= linux/amd64 linux/arm64
 CTF_TYPE ?= directory
 
 REPO_ROOT                                      := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))/../..
-VERSION                                        = $(shell go run ../../api/version/generate/release_generate.go print-rc-version $(CANDIDATE))
+ifeq ($(VERSION),)
```

Comment:
I am not that in to `Makefile` syntax, but I would guess that `Version ?= $(...)` should produce the same result as the condition

Reply:
See `6.2.4 Conditional Variable Assignment` in https://www.gnu.org/software/make/manual/make.html

Reply:
https://github.com/open-component-model/ocm/pull/1124/files#diff-b0b6d9a4b18557276003816e5b6516ee6e07edbd1bf823505e9127530ec8917dR14-R15

Reply:
#trustmeitried

Reply:
From 6.2.4

> Note that a variable set to an empty value is still defined, so ‘?=’ will not set that variable.